web/src/engine/websites/decorators/Common.ts

```diff
@@ -471,5 +471,9 @@ export async function GetTypedData(buffer: ArrayBuffer): Promise<Blob> {
     if (bytes[0] === 0x42 && bytes[1] === 0x4D) {
         return new Blob([bytes], { type: 'image/bmp' });
     }
+    // AVIF [00 00 00 20 66 74 79 70 61 76 69 66 ] ' ftypavif'
+    if (bytes[8] === 0x61 && bytes[9] === 0x76 && bytes[0xA] === 0x69 && bytes[0xB] === 0x66) {
```

Comment:
I think its easier to read and understand when using decimal indices for the `byte` array instead of hexadecimal numbers